clients/src/test/java/org/apache/kafka/clients/consumer/internals/StreamsMembershipManagerTest.java

```diff
@@ -781,43 +889,6 @@ private void setupStreamsAssignmentInterfaceWithTwoSubtopologies(final String su
         );
     }
 
-    private void setupStreamsAssignmentInterfaceWithTwoConcatenedSubtopologies(final String subtopologyId1,
-                                                                               final String topicName1,
-                                                                               final String subtopologyId2,
-                                                                               final String topicName2) {
-        when(streamsAssignmentInterface.subtopologyMap()).thenReturn(
-            mkMap(
-                mkEntry(
-                    subtopologyId1,
-                    new StreamsAssignmentInterface.Subtopology(
-                        Set.of(topicName1),
-                        Collections.emptySet(),
-                        Collections.emptyMap(),
-                        Collections.emptyMap(),
-                        Collections.emptyList()
-                    )
-                ),
-                mkEntry(
-                    subtopologyId2,
-                    new StreamsAssignmentInterface.Subtopology(
-                        Set.of(topicName2),
-                        Collections.emptySet(),
-                        mkMap(mkEntry(
-                            topicName2,
-                            new StreamsAssignmentInterface.TopicInfo(
-                                Optional.empty(),
-                                Optional.empty(),
-                                Collections.emptyMap()
-                            )
-                        )),
-                        Collections.emptyMap(),
-                        Collections.emptyList()
-                    )
-                )
-            )
-        );
-    }
-
```

Comment:
We removed an assignment with 2 subtopologies - will we add it back at some point or is present in another form?

Reply:
Dunno.
For now, I did not see value in also having tests with this setup.